recipe/run_test.py

```diff
@@ -1,16 +1,15 @@
-
 # just load libfreexl using ctypes
 import os
 import sys
 import ctypes
 
-if sys.platform == 'win32':
-    libfreexl = ctypes.CDLL('freexl.dll')
-elif sys.platform == 'darwin':
+if sys.platform == "win32":
+    libfreexl = ctypes.CDLL("%PREFIX%\Library\\bin\\freexl.dll")
```

Comment:
@ocefpaf I think this test is missing an `os.path.expandvars` (like the Unix ones have)

Reply:
Indeed, sorry. But it was passing before that test and I'm not sure we even need the full path.